pkgs/top-level/aliases.nix

```diff
@@ -219,6 +222,7 @@ mapAliases ({
   man_db = man-db; # added 2016-05
   manpages = man-pages; # added 2015-12-06
   mariadb-client = hiPrio mariadb.client; #added 2019.07.28
+  mbedtls_1_3 = throw "End of life, see https://tls.mbed.org/kb/how-to/upgrade-2.0"; # added 2019-12-08
```

Comment:
Can you add `mbedtls_1_3 is end of life, ...`.
I've noticed it's frequently not apparent when you encounter the throw what attr it is.

Reply:
Right. Done in a8df1b12307b41a0bc1bbc1598be8bb882c3a41e